pkg/alert/alert.go

```diff
@@ -354,7 +405,7 @@ func (s *Sender) Send(ctx context.Context, alerts []*Alert) {
 	}
 
 	s.dropped.Add(float64(len(alerts)))
-	level.Warn(s.logger).Log("msg", "failed to send alerts to all alertmanagers", "alerts", string(b))
+	level.Warn(s.logger).Log("msg", "failed to send alerts to all alertmanagers")
```

Comment:
A very small thing but maybe there is still some worth in logging how many alerts we have failed to send?

Reply:
done